plugins/introduction.rst

```diff
@@ -4,11 +4,18 @@ Introduction
 Install
 -------
 
-Install the plugin client in your project with Composer_:
+The plugin client and the core plugins are available in the `php-http/client-common`_ package:
```

Comment:
i think here would be the other place to have versionadded and say that this is only the case since 1.1 and for 1.0 you need to require php-http/plugins.